src/components/transactions/Borrow/BorrowModalContent.tsx

```diff
@@ -222,7 +224,7 @@ export const BorrowModalContent = ({
 
   return (
     <>
-      {symbol === 'DAI' && <SpkAirdropNote />}
+      {symbol === 'DAI' && showAirdropInfo && <SpkAirdropNote />}
```

Comment:
what do you think about inlining this "showAirdropInfo" check in the component it component itself? I think it will make maintenance easier as we just need to check it in a single place.